config/initializers/filter_parameter_logging.rb

```diff
@@ -0,0 +1,6 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Whats this file about?

Reply:
We can store sensitive parameters such as passwords, api tokens, secrets, etc. inside `Rails.application.config.filter_parameters` to filter them out by displaying them as `[FILTERED]` on logs.

Reply:
You don't `store` per se. You set the config.